lib/diagnostic.rb

```diff
@@ -35,8 +48,9 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+def dave=(name: "Dave", age: 32, location: "Ohio")
+  dave.location = "Somerville"
+Response.dave = dave.location
 ##
```

Comment:
make sure you use the Class.new syntax when instantiating dave as a new person. 
Also, dave doesn't need the def before it.
So `dave = Person.new("Dave", 32, "Ohio")`
Great job changing the location! -.5